build.rs

```diff
@@ -591,8 +578,11 @@ fn cc(file: &Path, ext: &str, target: &Target, include_dir: &Path, out_file: &Pa
     if target.os.as_str() == "macos" {
         // ``-gfull`` is required for Darwin's |-dead_strip|.
         let _ = c.flag("-gfull");
+        let _ = c.flag("-fPIC");
+        let _ = c.flag("-Wl,-dead_strip");
     } else if !compiler.is_like_msvc() {
         let _ = c.flag("-g3");
+        let _ = c.flag("-Wl,--gc-sections");
```

Comment:
`-Wl,--gc-sections` is no longer given to MSVC, but otherwise the effect is equivalent to before.